src/pineko/fonll.py

```diff
@@ -239,18 +233,30 @@ def produce_fonll_recipe(fonll_fns, damp):
     """Produce the different theory cards according to which FONLL is asked for."""
     fonll_recipe = []
     is_mixed_or_damp = fonll_fns in MIXED_ORDER_FNS or damp != 0
-    fns_list = MIXED_FNS_LIST if is_mixed_or_damp else FNS_LIST
-    nfff_list = MIXED_NFFF_LIST if is_mixed_or_damp else NFFF_LIST
-    parts_list = MIXED_PARTS_LIST if is_mixed_or_damp else PARTS_LIST
+    fns_list = (
+        np.array(MIXED_FNS_CONFIG).transpose().tolist()[0]
+        if is_mixed_or_damp
+        else np.array(FNS_CONFIG).transpose().tolist()[0]
+    )
+    nfff_list = (
+        np.array(MIXED_FNS_CONFIG).transpose().tolist()[1]
+        if is_mixed_or_damp
+        else np.array(FNS_CONFIG).transpose().tolist()[1]
+    )
+    parts_list = (
+        np.array(MIXED_FNS_CONFIG).transpose().tolist()[2]
+        if is_mixed_or_damp
+        else np.array(FNS_CONFIG).transpose().tolist()[2]
+    )
     for fns, nfff, po, part in zip(
         fns_list, nfff_list, produce_ptos(fonll_fns, is_mixed_or_damp), parts_list
     ):
```

Comment:
Now you're transposing and zipping here (which are inverse to each other), but this can be simplified: add to your scheme definitions a new field `delta_pto` (which can be 1 or 0) and then you can do a simple iteration here and put `"PTO": base_pto + delta_pto`

We could/should maybe even put a dataclass `FNSSchemeConfig` (or similar) - and we can make factors (i.e. minuses and damping part of it)

Reply:
actually, damping depends on the grid so that could be left apart - but minuses are universal and can be attributed to a config

Reply:
I followed your suggestion (more or less). Let me know if you like it more now :) 
[7e385a3](https://github.com/NNPDF/pineko/pull/99/commits/7e385a36e70932544de4a75b8fd753f2169f7f3b)

Reply:
I'm sorry, but no :upside_down_face: (because you transposed again) - however, let me have a shot myself for it

Reply:
No wait, I never transposed now

Reply:
#131